CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+**September 5, 2023 [DESCW-1268](https://apps.itsm.gov.bc.ca/jira/browse/DESCW-1268)**
```

Comment:
just a bit of caution in the amount of items you add here in the changelog, not sure the intention is to give a step by step of all the code changes, more is not always better.

Reply:
fixed